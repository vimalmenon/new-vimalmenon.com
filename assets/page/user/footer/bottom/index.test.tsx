import "jsdom-global/register";
import React from "react";
import renderer from "react-test-renderer";
import FooterBottom from "./index";
import {shallow} from "enzyme";

describe("SocialMedia Component", ()=> {
    it("Snapshot Testing", () => {
        const component = renderer.create(<FooterBottom />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
        expect(true).toBe(true);
    });
});