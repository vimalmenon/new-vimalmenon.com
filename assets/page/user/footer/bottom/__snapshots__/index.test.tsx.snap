// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMedia Component Snapshot Testing 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-container-2"
  >
    <div
      className="makeStyles-section-3"
    >
      <a
        className="makeStyles-footerName-4"
        href="/"
      >
        VimalMenon.com
      </a>
      <span
        className="makeStyles-span-5"
      >
        Â© All Right Reserved - 2020
      </span>
    </div>
    <div
      className="makeStyles-section-3"
    >
      <span>
        v
        0.0.1
      </span>
    </div>
  </div>
</div>
`;
